### PR TITLE
Gather container's specific filenames in one place

### PR DESCRIPTION
**Pull request by Rachid-Koucha**
The names of the container's specific file/directory are hard coded in one place (lxccontainer.h) instead of repeating the strings everywhere in lxccontainer.c

 Signed-off-by: Rachid Koucha <rachid.koucha@gmail.com>

### CODE REVIEW

**Comment by brauner** [comment]
Hey, could you please sign-off your commits? :)

**Comment by Rachid-Koucha** [comment]
I signed them but after fixing a compilation error it seems no longer taken in account. And as usual for me, through github, it is tough to retrieve/fix the missing signed :-(

**Comment by brauner** [comment]
> I signed them but after fixing a compilation error it seems no longer taken in account. And as usual for me, through github, it is tough to retrieve/fix the missing signed :-(

Ok, let me fix it then.

File: src/lxc/lxccontainer.h [hunk 1]
```diff
@@ -65,6 +65,13 @@ enum {
 	LXC_SECCOMP_NOTIFY_MAX,
 };
 
+/* Container's specific file names */
+#define LXC_CONFIG_FNAME       "config"
+#define LXC_PARTIAL_FNAME      "partial"
+#define LXC_ROOTFS_DNAME       "rootfs"
+#define LXC_TIMESTAMP_FNAME    "ts"
+#define LXC_COMMENT_FNAME      "comment"
```
Comment (brauner) [inline src/lxc/lxccontainer.h]: Hm, you can't define this in `lxccontainer.h` this is a public header file.

File: src/lxc/lxccontainer.c [hunk 2]
```diff
@@ -4228,7 +4228,7 @@ static int do_lxcapi_snapshot(struct lxc_container *c, const char *commentfile)
 
 	strftime(buffer, 25, "%Y:%m:%d %H:%M:%S", &tm_info);
 
-	char *dfnam = alloca(strlen(snappath) + 1 + strlen(newname) + 1 + strlen(LXC_TIMESTAMP_FNAME) + 1);
+	dfnam = alloca(strlen(snappath) + 1 + strlen(newname) + 1 + strlen(LXC_TIMESTAMP_FNAME) + 1);
```
Comment (brauner) [inline src/lxc/lxccontainer.c]: Wait, how is that possible? There are no `alloca()` calls in master anymore... This is `must_realloc()` in master...

File: src/lxc/lxccontainer.h [hunk 3]
```diff
@@ -66,11 +66,11 @@ enum {
...
-#define LXC_CONFIG_FNAME       "config"
-#define LXC_PARTIAL_FNAME      "partial"
-#define LXC_ROOTFS_DNAME       "rootfs"
-#define LXC_TIMESTAMP_FNAME    "ts"
-#define LXC_COMMENT_FNAME      "comment"
+#define LXC_CONFIG_FNAME      "config"
+#define LXC_PARTIAL_FNAME     "partial"
+#define LXC_ROOTFS_DNAME      "rootfs"
+#define LXC_TIMESTAMP_FNAME   "ts"
+#define LXC_COMMENT_FNAME     "comment"
```
Comment (brauner) [inline src/lxc/lxccontainer.h]: This should not be a separate commit.

Comment (Rachid-Koucha) [reply src/lxc/lxccontainer.h]: If I can't modify lxccontainer.h, should I define the macro names at the top level of lxccontainer.c ?

Comment (brauner) [reply src/lxc/lxccontainer.h]: Or you put them in `macro.h`.

File: src/lxc/lxccontainer.c (hunk 2 above)
Comment (brauner) [reply src/lxc/lxccontainer.c]: Ok, Why are you changing this to `alloca()`? `alloca()` is banned from the codebase.
